article/adj-difference-matrix.tex

```diff
@@ -0,0 +1,19 @@
+\subsection{Generated adjacency matrix}
+In the following subsection the generated adjacency matrix of LightGCN and price aware recommendation are described.
+
+\subsection{LightGCN}
+For LightGCN the adjacency matrix is the size of $\textrm{number of users} + \textrm{number of items} \times \textrm{number of users} + \textrm{number of items}$.
+Then for every connection between item $i$ and user $u$ 1 is inserted.
+
+\subsection{Price-aware recommendation}
```

Comment:
Tror du misforstår det lidt.
hvis num_nodes = number of users + number of items + number of categories + number of prices
Så har adjacency matrixen størelsen: num_nodes X num_nodes

Det du beskriver med row og col er implementations specifikt hvordan man laver en sparse matrix i scipy. Tror ikke at det er nædvøndigt at beskrive

Reply:
størrelsen af row og col ahr ikke noget med størelsen af selve matrixen at gøre.
Så adjacency matrixen i lightGCN og PUP er faktisk lavet på samme måde. PUP har bare categories og prices med også